### DISCUSSION TITLE
Query Help

### DISCUSSION THREAD

**Discussion by nomanmurtaza786** [supabase/supabase#2120] [Questions] [answered]
Hi guys I want to get today's most-liked post. How can performance this query with superbase?
Of any other efficient way to doing this with supabase. 

select post_id, count(*) from vote_up_logs 
where vote_date > NOW() - INTERVAL '24 Hour'
group by coin_id 

Thanks

**Comment by inian** [comment, marked as answer]
Hi @nomanmurtaza786, you need to use post_id in your group by clause. We can help you out better if you provide a reproducible example and the error message
 
```
select post_id, count(*) from vote_up_logs
where vote_date > NOW() - INTERVAL '24 Hour'
group by post_id
```

**Reply by nomanmurtaza786** [reply]
Thanks @inian its sloved

**Reply by Raidus** [reply]
@nomanmurtaza786 can you post you full code snippet for supabase api? I can't make it work with the interval part
